updates/version.yaml

```diff
@@ -1,490 +1,525 @@
+#v3.2.0:
+#    - 'Feature Release'
+#    - 'Decreased PHP Requirement to 7.4 to re-support OC v2.2 installations.'
+#    - 'Added new navigation icons for catalogue and orders menus.'
+#    - 'Added new <grp>_<idx>_<action>_<name>.php migration file naming for a better overview.'
+#    - 'Added new enabled-state to currency + minor changes + tests.'
+#    - 'Added new enabled-state and additional title property to price category + minor changes + tests.'
+#    - 'Added new enabled-state to taxes + minor changes + tests.'
+#    - 'Added new enabled and default-states to payment methods + minor changes + tests.'
+#    - 'Added new enabled and default-states to shipping methods + minor changes + tests.'
+#    - 'Added new enabled-state to order states + minor changes.'
+#    - 'Added a new IsStates database model trait, to support is_default and is_enabled behaviors.'
+#    - 'Added a new PriceBag totals construct and handling.'
+#    - 'Added a new event to manipulate shipping country on FilteredTaxes trait, thanks to @Cryden.'
+#    - 'Updated console commands and database seeders.'
+#    - 'Updated demonstration content (+ add support for german translations).'
+#    - 'Updated available rounding options on Currency model to support extensions.'
+#    - 'Catch errors on HashIds trait + add typings.'
+#    - 'General code cleanup, especially on backend controllers and models.'
+#    - 'Replaced migration seeder in favor of Laravels / Octobers database seeders.'
+#    - 'Replaced old ReorderController behavior with ListController configuration.'
+#    - 'Replaced VuePress documentation with VitePress + refactored introduction pages.'
+#    - 'Fixed missing quickCheckout pages on GeneralSettings > Checkout page select field.'
+#    - 'Fixed missing translation strings.'
+#    - 'Fixed show OrderProducts on Orderlist -> ProductList component using the same currency as order itself.'
+#    - 'Fixed PHPUnit testing environment and tests.'
+#    - 'Fixed Wishlist component with no id parameter set.'
+#    - 'Fixed faulty empty-check that prevented indexing at 0 (int and string) values, thanks to @xyz1123581321.'
+#    - 'Fixed out of stock message even if stock was available, thanks to @xyz1123581321.'
+#    - 'Fixed wrong indexed value key on PropertyValue using color types, thanks to @toome123.'
+#    - 'Fixed wrong taxes calculation when discounts has been applied.'
+#    - 'Fixed wrong taxes calculation when different discounts has been applied to products and shipping costs.'
+#    - 'Fixed bad SQL error when addresses has been added to a customer profile on the backend.'
+#    - 'Fixed duplication mess on products with variants.'
+#    - 'Fixed possibility to take-over email address on CustomerProfile component, thanks to @cyril-design.'
+#    - 'Fixed products / variants disappear on first page view after editing a product.'
+#    - 'Removed legacy v1 code.'
+#    - 030_01-update_system_plugin_history.php
```

Comment:
What is the purpose of this migration? It currently fails on my test installation:

> In 030_01-update_system_plugin_history.php line 74:
>
>   Migration file "030_01-add_product_variants_sort_order.php" on path "/user/code/oc-mall-plugin/updates" does not exist.

Reply:
Finally fixed this issue. Looks like v2 has been developed alongside v3, which resulted in different migration histories and (gladly just) one additional update-file. (Which, by the way, led to many errors when migrating from Mall v2.0.12 or higher to master / v3.)

However, to answer your question. This change was supposed to improve the overall view of the available migration files by using groups and a file index as a prefix. This pattern allows to simple identify which update-files belongs together, which simplifies backtracking migration / migrate:refresh issues.

Of course, I can undo this change if you wish.